.github/workflows/aml-nightly.yml

```diff
@@ -0,0 +1,98 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# ---------------------------------------------------------
+
+name: aml-nightly
```

Comment:
In the past, we got told by Marketing that we can't use AML, instead we need to use AzureML or azureml. Could you please change all the instances of `aml` by `azureml`?

Reply:
Changed all `aml` instances to `azureml` except azureml compute credentials and resource group names (will update those once we get the additional compute).